src/js/modules/infragistics.ui.editors.js

```diff
@@ -38,36 +38,168 @@
 	$.widget("ui.igBaseEditor", {
 		options: {
 			/* type="string|number|null" Gets/Sets how the width of the control can be set.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					width : 200
+				});
+
+				//Get
+				var width = $(".selector").%%WidgetName%%("option", "width");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "width", 200);
+				```
 				string The widget width can be set in pixels (px) and percentage (%).
 				number The widget width can be set as a number in pixels.
 				null type="object" will stretch to fit data, if no other widths are defined.
 			*/
 			width: null,
 			/* type="string|number|null" Gets/Sets how the height of the control can be set.
+				```
+				//Initialize
+				$(".selector").%%WidgetName%%({
+					height : 25
+				});
+
+				//Get
+				var height = $(".selector").%%WidgetName%%("option", "height");
+
+				//Set
+				$(".selector").%%WidgetName%%("option", "height", 25);
+				```
 				string The height can be set in pixels (px) and percentage (%).
 				number The height can be set as a number in pixels.
 				null type="object" will fit the editor inside its parent container, if no other heights are defined.
 			*/
 			height: null,
-			/* type="object" Gets/Sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.*/
+			/* type="object" Gets/Sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.
```

Comment:
if we want to have option reference here we need to copy paste the option and the comment in all the derived widgets, because the dataMode exists in almost every editor and we don't have a mechanism for templating the widget name within the comment, but only for the code snippet.